### PR TITLE
Add compare hooks for regex literals and character ranges

### PR DESCRIPTION
**Pull request by fmeum**

### CODE REVIEW

File: sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/RegexRoadblocks.java [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+import com.code_intelligence.jazzer.api.Jazzer;
+import com.code_intelligence.jazzer.api.MethodHook;
+import com.code_intelligence.jazzer.runtime.UnsafeProvider;
+import java.lang.invoke.MethodHandle;
+import java.util.WeakHashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import sun.misc.Unsafe;
+
+public final class RegexRoadblocks {
```
[the remark this answers is not in the document]
Comment (fmeum) [reply sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/RegexRoadblocks.java:43]: Done, added more comments throughout the code.

File: .bazelrc [hunk 2]
```diff
@@ -33,6 +33,10 @@ run:windows --noincompatible_strict_action_env
 # Since the toolchain is conditional on OS and architecture, set it on the particular GitHub Action.
 build:toolchain --//third_party:toolchain
 
+# Forward debug variables to tests
```
[the remark this answers is not in the document]
Comment (bertschneider) [reply .bazelrc:36]: Quite some time ago I saw a talk about it and always wanted to try it out. I add it to the ever growing list of things to look into :+1:

File: sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/RegexRoadblocks.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (bertschneider) [reply sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/RegexRoadblocks.java:43]: The comments are really thorough and good to understand, thx.

**Review by bertschneider — APPROVED** [review]
Really nice improvement! You made it looks simple :+1:

**Pull request merged**